src/assets/wise5/services/ideasSortingService.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+
+export type IdeaData = {
+  id: string;
+  text: string;
+  count: number;
+};
+
+@Injectable()
+export class IdeasSortingService {
+  constructor() {}
+
+  sortIdeasByCount(ideas: IdeaData[]): IdeaData[] {
```

Comment:
"Ideas" is redundant because the class is IdeasSortingService?
```suggestion
  sortByCount(ideas: IdeaData[]): IdeaData[] {
```